README.md

```diff
@@ -6,7 +6,7 @@
 
 # SPDX tools-golang
 
-tools-golang is a collection of Go packages intended to make it easier for
+`tools-golang` is a collection of Go packages intended to make it easier for
```

Comment:
I think it will be super helpful to link your usage repo somewhere in this README or as a comment in the root `/spdx` package as an example of how to use this new refactored version!

Reply:
The problem with the usage repo is it's just a branch that will probably get deleted. Once this is merged and I've updated Syft, I could have a follow-on to the README that links directly to the Syft usage but probably shouldn't put any "permanent" links before that.